app/build.gradle

```diff
@@ -33,7 +33,7 @@ android {
 
   defaultConfig {
     applicationId "com.mapzen.erasermap"
-    minSdkVersion 15
+    minSdkVersion 17
```

Comment:
Reverted, we don't need to change this value. In testing, I had layout param verbs that required this but then removed them.